tasks/section3.yml

```diff
@@ -271,15 +336,17 @@
       - rule_3.4.1
 
 - name: "SCORED | 3.4.1 | PATCH | Ensure TCP Wrappers is installed"
-  command: /bin/true
+  yum:
+      name=tcp_wrappers
+      state=installed
```

Comment:
This task needs to be in YAML literal form per https://github.com/ansible/ansible-lockdown/blob/master/CONTRIBUTING.md